roles/katello_repositories/tasks/staging_repos.yml

```diff
@@ -49,4 +49,4 @@
     gpgcheck: 0
   when:
     - katello_repositories_version != "nightly"
-    - katello_repositories_version is version('3.8', '<=')
+    - katello_repositories_version is version('3.8', '<=')
```

Comment:
Nit: newline

Reply:
whatever made this happen, I'll fix that.